xbmc/GUIInfoManager.cpp

```diff
@@ -4008,13 +4008,19 @@ const infomap fanart_labels[] =  {{ "color1",           FANART_COLOR1 },
 ///                  _boolean_,
 ///     todo
 ///   }
+///   \table_row3{   <b>`Skin.Font`</b>,
+///                  \anchor Skin_Font
+///                  _boolean_,
```

Comment:
Yup. The whole docs are a mess in this file, but no point in adding wrong *new* docs